CHANGELOG.md

```diff
@@ -127,6 +127,95 @@ case it elicits some feedback that requires changes.
 
 * [Video Services](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/video/README.md)
 
+### [Bigtable](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigtable/README.md)
+
+We introduced a [new constructor][modern-table-ctor] for `Table` which accepts a
+`DataConnection` instead of a `DataClient`. The `DataConnection` is a new
+interface that more closely matches the client surface of `Table`. Read more
+about `*Connection` classes in our
+[Architecture Design][architecture-connection] document.
+
+**What are the benefits of `DataConnection`?**
+
+The new API greatly simplifies mocking. Every `Table::Foo(..)` call has an
+associated `DataConnection::Foo(...)` call. This allows you to set expectations
+on the exact values returned by the client call. See
+[Mocking the Cloud Bigtable C++ Client][howto-mock-data-api] for a complete
+example on how to mock the behavior of `Table` with
+`bigtable_mocks::MockDataConnection`.
+
+The new `DataConnection` API offers more consistency across our libraries. It
+also enables the use of some common library features, such as our
+[`UnifiedCredentialsOption`][guac-dox]. Also, any new features will be added to
+the `DataConnection` API first.
+
+**Do I need to update my code?**
+
+No. If the benefits are not appealing enough, you do not need to update your
+code. All code that currently uses `DataClient` will continue to function as
+before. This includes uses of `testing::MockDataClient`.
+
+However, if you are using `testing::MockDataClient` to mock the behavior of
+`Table` in your tests:
+
+1. I am impressed, that is not easy to do.
+2. Be aware that we have announced our intention to remove classes derived from
+   `DataClient` on or around 2023-05. Your tests will break then.
+3. Please consider using `bigtable_mocks::MockDataConnection`. It will greatly
+   simplify your tests.
+
+**How do I update existing `DataClient` code?**
+
+```c++
+namespace gc = ::google::cloud;
+namespace cbt = ::google::cloud::bigtable;
+
+// An option which affects the `DataClient` / `DataConnection`.
+auto options = gc::Options{}.set<gc::GrpcNumChannels>(4);
+
+// Existing code...
```

Comment:
Maybe wrap these in a function?

```cc
void OldVersion() {
  auto connection = cbt::MakeDataConnection(options);
...
}

void NewCode() {
  auto connection = cbt::MakeDataConnection(...);
...
}
```

You may want to consider a simpler example first, this gives folks sticker shock.  Start with something that has no profiles and no app profile and then add introduce the additional complexity.